main.go

```diff
@@ -2,35 +2,68 @@
 package main
 
 import (
+	"fmt"
 	"gopkg.in/urfave/cli.v2"
 	"log"
 	"os"
 	"reflect"
+	"time"
 )
 
 var (
 	version = "dev"
 )
 
+type config struct {
+	delay   time.Duration
+	verbose bool
+	dryrun  bool
+}
+
 func main() {
+
+	// do not use "-v" to print the version
+	cli.VersionFlag = &cli.BoolFlag{
+		Name: "version", Aliases: []string{},
+		Usage: "print the version only",
+	}
+
 	app := &cli.App{
-		Name:      "terraform-state-mover",
-		Usage:     "refactoring Terraform code has never been easier",
-		Authors:   []*cli.Author{{Name: "Maximilian Bode", Email: "maxbode@gmail.com"}},
-		Action:    action,
-		UsageText: "terraform-state-mover [-- <terraform args>]",
+		Name:    "terraform-state-mover",
+		Usage:   "refactoring Terraform code has never been easier",
+		Authors: []*cli.Author{{Name: "Maximilian Bode", Email: "maxbode@gmail.com"}},
+		Action:  action,
+		Flags: []cli.Flag{
+			&cli.DurationFlag{
+				Name: "delay", Aliases: []string{"d"},
+				Usage: "Delay between terraform state mv calls. Helps to avoid rate-limits.",
+				Value: time.Second * 2,
```

Comment:
I'm inclined to use a default of zero seconds so as users with providers without rate limiting are not slowed down unnecessarily. What do you think?

Reply:
makes sense. Changed the default to "0s" (i.e. no delay). I added a hint to the README.md for GCP provider user where the delay flag is often necessary.